docker-compose.yml

```diff
@@ -112,6 +112,34 @@ services:
       - ipfs
     command: /wait-for-it.sh -h rabbit -p 5672 -t 30 -- /wait-for-it.sh -h mongo -p 27017 -- npm start
 
+  poet-node_blockchain-writer:
```

Comment:
Maybe use dashes everywhere for consistency? `poet-node-blockchain-writer`

Reply:
That's fine, I was thinking that the 'lodash' was a way to separate the higher order name from the individual module.